Source/WebCore/platform/ios/VideoFullscreenInterfaceAVKit.mm

```diff
@@ -483,6 +483,10 @@ - (void)removeObserver
 
 - (void)tryToStartPictureInPicture MY_NO_RETURN
 {
+#if PLATFORM(WATCHOS)
```

Comment:
Nit: Worth checking with the media folks if this should actually be some "does have Picture in Picture", which we tend to prefer over platform checks.

Reply:
This is how the check is done in other parts of this file (for example in `-startPictureInPicture`, just below).

I don't really understand why clang started catching this noreturn violation now, as it seems like it's always been the case.